modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/CreateContentCommand.java

```diff
@@ -108,6 +96,7 @@ private Content doExecute()
             .xDataService( this.xDataService )
             .partDescriptorService( this.partDescriptorService )
             .layoutDescriptorService( this.layoutDescriptorService )
+            .contentDataSerializer( this.translator.getContentDataSerializer() )
```

Comment:
maybe doe to this change we'll start to save empty regions into data. Need to verify